src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -274,19 +279,28 @@ public void runAppCfgCommand(List<String> args) throws ProcessRunnerException {
   }
 
   /**
-   * Returns the version of the Cloud SDK installation. Unlike other methods in this class that call
-   * gcloud, this method always uses a synchronous ProcessRunner and will block until the gcloud
-   * process returns.
+   * Returns the version of the Cloud SDK installation.
    *
-   * @throws ProcessRunnerException when process runner encounters an error
+   * @throws CloudSdkOutOfDateException if the Cloud SDK is too out of date to determine its version
```

Comment:
[Here](https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/276/files#diff-562793416b98626facf7820af2a81c16R286) is what I went with. Looks good?